config/locales/en/course/leaderboard.yml

```diff
@@ -5,3 +5,7 @@ en:
       title: :'course.leaderboard.index.header'
       index:
         header: 'Leaderboard'
+      show:
+        level_header: 'By Level'
+        level: 'Level %{level_number}'
```

Comment:
might wanna factor out to the common file? or a common course translations file?